package.json

```diff
@@ -2,56 +2,43 @@
   "name": "bitski",
   "description": "Bitski JavaScript SDK",
   "license": "MIT",
-  "main": "dist/bitski.umd.js",
-  "module": "dist/bitski.es5.js",
+  "main": "dist/lib/bitski.js",
   "typings": "dist/types/bitski.d.ts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/BitskiCo/bitski-js"
+    "url": "https://github.com/BitskiCo/bitski-js-sdk"
   },
-  "version": "0.1.10",
+  "version": "0.2.0",
```

Comment:
do we want to bump the version yet? or should we wait until we do a release branch?

Reply:
Good point, I will revert.